pyo3-macros-backend/src/method.rs

```diff
@@ -302,19 +308,26 @@ impl<'a> FnSpec<'a> {
                 .collect::<Result<_>>()?
         };
 
+        let signature = if let Some(signature) = signature {
```

Comment:
Is it currently possible to specify both `#[pyo3(signature)]` and `#[args()]` with signature overriding everything in args? If so, I think that should become a hard error.

Reply:
Found the answer further down below. Sorry for the noise.

Reply:
Actually good to call this out - I had made it a hard error for `#[pyfunction]` but forgotten to do the same for `#[pymethods]`. I've pushed a fix!